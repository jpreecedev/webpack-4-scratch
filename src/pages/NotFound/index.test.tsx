import React from "react"

import { NotFound } from "."

describe("<NotFound /> tests", () => {
  it("should match snapshot", () => {
    expect(<NotFound />).toMatchSnapshot()
  })
})
