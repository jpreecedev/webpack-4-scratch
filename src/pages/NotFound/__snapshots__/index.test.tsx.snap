// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> tests should match snapshot 1`] = `<NotFound />`;
